meetings/2021-06-16.md

```diff
@@ -0,0 +1,151 @@
+
+Aaron: @bblfish:matrix.org you are conflating OIDC and OAuth2. We're talking about OAuth2, not OIDC.
+
+Henry: one does not need in depth knowledge of either technology to understand its limitations, which arise directly out of not accepting the Semantic web or linked data principles. By deliberately ignoring these technologies, they will have had to come up with very awkward answers to problems that would have had clean mathematical/logical solutions otherwise.
+
+Henry: You say we don't need the Guard. Do we now throw authz frameworks out we have been building?
+
+Aaron: Access Control is completely orthogonal to this. WAC and ACP are data models. UMA is a protocol.
+
+Henry: WAC is actually a protocol:  The web is designed so you can get to a resource from anywhere. So you reach resource get a 401 and follow the link header to the Access Control Resource. Then decide which credentials to send, if any, and the server can authorize the request.
+
+Aaron: Both WAC and ACP are orthogonal. The Guard on the server level runtime is no longer needed because all they need to know is: Is the token valid for that request? The authorization server goes to `/foo/bar`, gets a 401. (??? bblfish: something missing here) UMA defines you take the value to your Authz server which gives you a token for that particular request and that's where ACP or WAC gets evaluated.
+
+Henry: How does the authorization server on a different POD know what the access control rules are to access a resource there? 
+
+Aaron: A pod has a specific authorization server.
+
+Henry: An example: I am bblfish.net and want to go to a resource `R` on justinb.net which allows access only to friends of a friend. Someone needs to look at the ACR for `R` to find out whether I have an identity in my wallet that satisfies that rule there. How does that work?
+
+Aaron: Can I suggest that we focus on UMA instead of OAuth 101. 
+
+Elf: I shouldn't expect that we clarify everything over the call. Maybe do some sequence diagram to showcase how it works. Maybe after the meeting? When we talk about the guard and who is enforcing the access rules, the resource server with that token is responsible for answering to that request. Getting an access token might let you access more than one resource. It would be good to break it down with one example.
+
+Henry: +1 agree on having a use case that covers how this would work on a decentralised system.
+
+Justin: +1 detailed use case.
+There is a pretty active thread since last week in the CCG related to the VC http api; and looking at which authorization protocol it should support. The dialog was very informative to this discussion. Everybody agreed that GNAP had a lot of potential, part of the future, not fully baked. The CCG can't put forward a mature spec.
```

Comment:
```suggestion
There is a pretty active thread since last week in the CCG related to the VC-HTTP-API, and looking at which authorization protocol it should support. The dialog was very informative to this discussion. Everybody agreed that GNAP had a lot of potential, part of the future, not fully baked. The CCG can't put forward a mature spec.
```